buildSrc/src/main/kotlin/CommonConfig.kt

```diff
@@ -50,7 +50,7 @@ fun Project.applyCommonConfiguration() {
 
     plugins.withId("java") {
         the<JavaPluginExtension>().toolchain {
-            languageVersion.set(JavaLanguageVersion.of(8))
+            languageVersion.set(JavaLanguageVersion.of(11))
```

Comment:
Preferably add a JDK vendor, like open jdk, if Java 11 is not installed allowing gradle to grab one.

Reply:
Isn't the vendor, by default, AdoptOpenJDK?

Reply:
> Isn't the vendor, by default, AdoptOpenJDK?

Yes. The point was to add a field to exchange the jvm vendor with ease if desired.